q2_diversity/plugin_setup.py

```diff
@@ -38,15 +38,27 @@
                  'and exploring community alpha and beta diversity through '
                  'statistics and visualizations in the context of sample '
                  'metadata.'),
-    short_description='Plugin for exploring community diversity.'
+    short_description='Plugin for exploring community diversity.',
+    citation_text=('Unweighted UniFrac: '
+                   'Lozupone and Knight 2005 Appl Environ Microbiol; DOI: '
+                   '10.1128/AEM.71.12.8228-8235.2005.\n'
+                   'Weighted UniFrac: '
+                   'Lozupone et al. 2007 Appl Environ Microbiol; DOI: '
+                   '10.1128/AEM.01996-06.\n'
+                   'Variance adjusted UniFrac: '
+                   'Chang et al. BMC Bioinformatics 2011 '
+                   'https://doi.org/10.1186/1471-2105-12-118.\n'
+                   'Generalized UniFrac: '
+                   'Chen et al. 2012 Bioinformatics; DOI: '
+                   '10.1093/bioinformatics/bts342')
 )
 
 plugin.methods.register_function(
     function=q2_diversity.beta_phylogenetic,
     inputs={'table': FeatureTable[Frequency],
             'phylogeny': Phylogeny[Rooted]},
     parameters={'metric': Str % Choices(beta.phylogenetic_metrics()),
-                'n_jobs': Int},
+                'n_jobs': Int % Range(1, None)},
```

Comment:
This is minor, and ultimately whatever fits better into the surrounding terminology is preferred, but it is more correct to describe this as threads.

Reply:
Jobs is probably better for now, since it sounds like this is actually CPU threads, and not a process thread.

Reply:
It's process threads (i.e., `pthread`) that we bind to cores to reduce thread hopping and to leverage NUMA's default local allocation policy.

Reply:
When I wrote this, I knew it was misleading, but carried on since it matches beta_phylogenetic. Would probably be nice to have a standard designation for "workers" all throughout QIIME.